draw_app/app.py

```diff
@@ -1,11 +1,31 @@
 import json
-from flask import Flask
+from flask import Flask, request
 from drawer.draw import Drawer
+from points_container.PointsContainer import PointsContainer
 
 app = Flask(__name__)
 
-@app.route('/draw/rest/')
+@app.route('/draw/rest')
 def draw():
-    drawer = Drawer(['Kaju', 'Adi', 'Mariaczi', 'Karol', 'Sito'])
-    new_draw = drawer.draw()
+    new_draw = global_drawer.draw()
     return json.dumps(new_draw)
+
+@app.route('/draw/rest/addPoints', methods=['POST'])
+def addPoinstByName():
+    if request.method == 'POST':
+        """ Example usage:
+            curl -d '{"player":"Kaju", "player2":"Malcin"}' -X POST http://localhost:5000/draw/rest/addPoints
```

Comment:
No change needed here, but check out https://httpie.org/, it's much more friendly for using HTTP in the command line

Reply:
E.g. the example post method would look like:
`http localhost:5000/draw/rest/addPoints player=Kaju player2=Malcin`